### PR TITLE
fix(cli): fixing microservice generation issues

### PR DESCRIPTION
**Pull request by yeshamavani**
gh-00

## Description

fixing microservice generation issues.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/cli/src/generators/microservice/index.ts [hunk 1]
```diff
@@ -570,26 +581,48 @@ export default class MicroserviceGenerator extends AppGenerator<MicroserviceOpti
     );
   }
 
-  private _addMigrationScripts() {
+  private _addMigrationScripts(flag: boolean) {
```
Comment (akshatdubeysf) [inline packages/cli/src/generators/microservice/index.ts:584]: name the variable better

Comment (akshatdubeysf) [reply packages/cli/src/generators/microservice/index.ts:584]: what is the purpose of the flag?

Comment (yeshamavani) [reply packages/cli/src/generators/microservice/index.ts:584]: when the migrations are created for the first time scrips are added directly through templates and only for the subsequent times done here.

Comment (yeshamavani) [reply packages/cli/src/generators/microservice/index.ts]: changed the name

**Review by akshatdubeysf — APPROVED** [review]

**Pull request closed without merging**